package.json

```diff
@@ -14,10 +14,12 @@
     "add-asset-webpack-plugin": "1.0.0",
     "babel-loader": "8.1.0",
     "clipboard": "2.0.6",
+    "codemirror": "5.58.2",
```

Comment:
Maybe you want to remove this explicit dependency. Easymde already depends on it and if those versions start to differ, it might be possible that we load two versions of codemirror otherwise. (Actually 3 versions if you count the lazy-loaded files in vendor).

Reply:
```suggestion
```